engines/web/README.rdoc

```diff
@@ -0,0 +1,3 @@
+= Web
+
+This is the rails engine for the WEB OFN domain.
```

Comment:
This would be a good place to a wiki page explaining why we use engines and how they are organised. Or maybe a readme in engines?

Reply:
maybe the readme pointing to the wiki.
I'll do that.